experimental/GModule/Brueckner.jl

```diff
@@ -442,11 +451,18 @@ function lift(C::GModule, mp::Map)
       end
       return d
     end
+    @show [pro[i](epi) for i=1:ngens(G)]
     l= [GMtoGG(reduce(gen(G, i)), pro[i](epi)) for i=1:ngens(G)]
-#    @show map(order, l), order(prod(l))
+    @show map(order, l), order(prod(l))
+    global last_x = l
 #    @show map(order, gens(G)), order(prod(gens(G)))
 
-    h = hom(G, GG, gens(G), l)
+    h = try
+          hom(G, GG, gens(G), l)
```

Comment:
I think the generators for the domain can be omitted and then default to `gens(G)`. So does this also work?
```suggestion
          hom(G, GG, l)
```